components/resources/brave_vpn_strings.grdp

```diff
@@ -118,11 +132,19 @@
   <message name="IDS_BRAVE_VPN_SUPPORT_TICKET_FAILED" desc="Label to show failed response from support ticket creation request">
     Failed to create support ticket, check your connection.
   </message>
+  <message name="IDS_BRAVE_VPN_SELECT_YOUR_SERVER" desc="Label to header text of server selection panel">
+    Select your server
+  </message>
+
+  <message name="IDS_BRAVE_VPN_SELECT_PANEL_BACK_BUTTON_ARIA_LABEL" desc="Text for back button in select panel">
+    Go to main
+  </message>
```

Comment:
Do we need to use a separate string as `IDS_BRAVE_VPN_ERROR_PANEL_BACK_BUTTON_ARIA_LABEL`? I think it might not cause more translation work as we might be collapsing to unique strings, but it seems like it will always neccessarily be the same since it's the "Go to main" button?